products/chef-client.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Chef Infra Client
```

Comment:
The file should get updated to match this product title as well

Reply:
updated